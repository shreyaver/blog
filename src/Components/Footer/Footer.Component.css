footer {
    background-color: black;
    color: white;
    padding-top: 20px;
    padding-bottom: 20px;
}
.footerContent {
    width: 70%;
    min-width: 300px;
    margin: auto;
}