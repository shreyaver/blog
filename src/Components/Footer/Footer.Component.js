import React from 'react';
import './Footer.Component.css';

const Footer = (props) => {
    return (
        <footer>
        <div className="footerContent">
            <small>&copy;artifact.com 2019</small>
        </div>
        </footer>
    )
}
export default Footer;