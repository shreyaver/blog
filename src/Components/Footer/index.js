import React from 'react';
import './Footer.css';

const Footer = (props) => {
    return (
        <footer>
        <div className="footerContent">
            <small>&copy;artifact.com 2019</small>
        </div>
        </footer>
    )
}
export default Footer;