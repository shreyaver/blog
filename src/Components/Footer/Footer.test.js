import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '.';

describe('Footer', () => {
  it('renders without crashing', () => {
    const footer = renderer.create(<Footer />).toJSON();
    expect(footer).toMatchSnapshot();
  });
});
