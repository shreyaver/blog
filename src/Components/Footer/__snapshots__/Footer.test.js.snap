// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders without crashing 1`] = `
<footer>
  <div
    className="footerContent"
  >
    <small>
      Â©artifact.com 2019
    </small>
  </div>
</footer>
`;
